plus/tool/src/main/webapp/WEB-INF/templates/form.html

```diff
@@ -182,6 +182,21 @@ <h1><span th="#{plus.tool.tenant.title}">Tenant:<span>
     let inp = document.getElementById(id+'-boolean');
     inp.value = checked ? 'true' : 'false';
 }
+</script>
+
+<script th:inline="javascript">
+    function validateTenantForm() {
+        let issuerInput = document.getElementById("issuer");
```

Comment:
You can use `const` instead of `let` since their values won't be reassigned. I think.

Reply:
Good point! I will use ```const```.